lib/core/jobs/consumer.rb

```diff
@@ -37,7 +37,7 @@ def initialize(scheduler:,
       end
 
       def subscribe!(index_name:)
-        Utility::Logger.info("Starting a new consumer for #{@index_name} index")
+        Utility::Logger.info("Starting a new consumer for #{index_name} index")
 
         @index_name = index_name
```

Comment:
Is it okay to call `@index_name` before it has been assigned the value from the method argument? At a glance it looks like the memoised `@index_name` could be `nil` or contain an old value.

Reply:
AH I was reading the diff wrong, maybe I'm colourblind :facepalm:

Reply:
It won't complain but it does print empty string, which I don't think is what it intend to do.